Skin_surface_3/include/CGAL/Skin_surface_quadratic_surface_3.h

```diff
@@ -151,44 +159,47 @@ class Skin_surface_quadratic_surface_3 {
 
   /// Construct the intersection point with the segment (p0,p1)
   template <class Input_point>
-  Input_point to_surface(const Input_point &p0, const Input_point &p1) {
+  Input_point to_surface(const Input_point &p0, const Input_point &p1)
+  {
     // NGHK: We use the kernel trick again: DOCUMENT!!!!
     typedef Cartesian_converter<typename Input_point::R, K> Converter;
     Converter conv;
 
     Input_point pp0 = p0, pp1 = p1, mid;
     double sq_d = to_double(squared_distance(pp0,pp1));
-    
+
     if (value(conv(pp1)) < value(conv(pp0))) {
       std::swap(pp0, pp1);
     }
-    
+
     while (sq_d > 1.e-10) {
       mid = midpoint(pp0,pp1);
       if (value(conv(mid)) > 0) {
-	pp1 = mid;
+        pp1 = mid;
       } else {
-	pp0 = mid;
+        pp0 = mid;
       }
       sq_d /= 4;
     }
     return midpoint(pp0,pp1);
   }
+
 private:
   //template <class Input_point>
-  Vector compute_gradient(Point const &x) {
-    
+  Vector compute_gradient(Point const &x)
+  {
     FT vx = x.x() - p.x();
     FT vy = x.y() - p.y();
     FT vz = x.z() - p.z();
 
     return Vector(2*Q[0]*vx + Q[1]*vy + Q[3]*vz,
-		  Q[1]*vx + 2*Q[2]*vy + Q[4]*vz,
-		  Q[3]*vx + Q[4]*vy + 2*Q[5]*vz);
+                  Q[1]*vx + 2*Q[2]*vy + Q[4]*vz,
+                  Q[3]*vx + Q[4]*vy + 2*Q[5]*vz);
   }
-	
-  FT Q[6]; 
-  Point p; 
+
+  int dim;
```

Comment:
I moved it to be with the other members of the class, at the bottom (instead of being lost in the middle of functions).